openedx_events/tests/test_consume_events_command.py

```diff
@@ -0,0 +1,51 @@
+"""
+Tests for consume_events command.
+"""
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from openedx_events.management.commands.consume_events import Command
+
+
+class TestCommand(TestCase):
+    """
+    Tests for the consume_events management command
+    """
+
+    @patch('openedx_events.tooling.OpenEdxPublicSignal.get_signal_by_type', return_value="test-signal")
+    @patch('openedx_events.management.commands.consume_events.make_single_consumer', autospec=True)
+    def test_consumer_call_with_only_required_args(self, mock_make_consumer, _):
+        call_command(Command(), topic=['test'], group_id=['test'], signal=['test-signal'])
+        mock_make_consumer.assert_called_once_with(topic='test', group_id='test', signal='test-signal')
+
+    @patch('openedx_events.tooling.OpenEdxPublicSignal.get_signal_by_type', return_value="test-signal")
+    @patch('openedx_events.management.commands.consume_events.make_single_consumer', autospec=True)
+    def test_consumer_call_with_extra_args(self, mock_make_consumer, _):
```

Comment:
can we add docstrings here? you can follow the format of tests like this one: https://github.com/openedx/openedx-events/blob/main/openedx_events/tests/test_tooling.py#L94-L99